servers/audio/audio_stream.cpp

```diff
@@ -89,6 +113,13 @@ void AudioStreamPlayback::_bind_methods() {
 	GDVIRTUAL_BIND(_seek, "position")
 	GDVIRTUAL_BIND(_mix, "buffer", "rate_scale", "frames");
 	GDVIRTUAL_BIND(_tag_used_streams);
+
+	ClassDB::bind_method(D_METHOD("get_raw_audio", "rate_scale", "frames"), &AudioStreamPlayback::get_raw_audio);
+	ClassDB::bind_method(D_METHOD("start", "p_from_pos"), &AudioStreamPlayback::start_playback);
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("start", "from_pos"), &AudioStreamPlayback::start_playback, DEFVAL(0.0));
```
No prefix for bind arguments, also match the default arguments of the binding

Reply:
Done, thanks for the feedback